src/components/molecules/CategoryPost/index.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <form @submit.prevent="addCategory">
+  <form class="category-form" @submit.prevent="addCategory">
```

Comment:
[imo]

上述した内容で修正してもらえればこちらの記述は削除できるかと思います。

```suggestion
  <form @submit.prevent="addCategory">
```